savoirfairelinux/sous-chef
# Fixed #685: Added functionality to remove client status

## Fixes # aaboffill

### Changes proposed in this pull request:

* Changed `ForeignKey` relationship between `ClientScheduledStatus` class to `OneToOneField`, and to handle this relations as a `pair` 
* Implemented a functionality to remove the client scheduled status. 
* Included and fixes tests.

### Status


### How to verify this change

* Go to the `Client` details and select the link `Status`.
* If the Scheduled Status list is empty, we can add a new Scheduled Status in the top right of the page.
* At the last column of the Scheduled Status list, we can see the button `Remove`

### Deployment notes and migration


### New translatable strings
